src/js/view/components/LogViewer.js

```diff
@@ -257,6 +259,8 @@ const LogViewer = props => {
           Math.round(logSize - scrollPosition),
           props.nrOfLinesInViewer
         );
+        let logSizeMinusScrollPos = Math.round(logSize - scrollPosition);
+        console.log({ logSizeMinusScrollPos });
```

Comment:
Found a couple of console.logs that could probably be removed, here and on row 17.